shared/chat/conversation/attachment-popup/index.native.js

```diff
@@ -32,15 +32,10 @@ const AttachmentPopup = ({message, detailsPopupShowing, isZoomed, onCloseDetails
 
   if (!previewType || previewType === 'Other') {
     return (
-      <PopupDialog onClose={onClose} fill={true} styleContainer={{
+      <Box style={{
         ...globalStyles.flexBoxColumn,
+        ...globalStyles.fillAbsolute,
         backgroundColor: globalColors.white,
-        borderRadius: 0,
-        bottom: 0,
-        left: 0,
-        position: 'absolute',
-        right: 0,
-        top: 20,
```

Comment:
These styles were causing an extra gap at the top of the screen where the PopupDialog background was visible. Since we weren't intending the PopupDialog background to display at all, a Box component is sufficient here.